docker-compose.sierraleone.yml

```diff
@@ -0,0 +1,22 @@
+version: '3'
+services:
+  postgres-db:
+    container_name: database
+    image: pgracio/dhis2-db:2.27-sierra-leone
+    restart: always
```

Comment:
I think that this should be `on-failure` or `unless-stopped` so that the container doesn't keep restarting itself even when stopped.

Reply:
@bausmeier I have set all the containers to `always`, I'm thinking I update them all, unless is there a reason to only be updating postgres?

Reply:
Doing all of them makes sense 👍